source/material_model/rheology/composite_visco_plastic.cc

```diff
@@ -256,6 +323,11 @@ namespace aspect
                            Patterns::Bool (),
                            "Whether to include Peierls creep in the rheological formulation.");
 
+        prm.declare_entry ("Include Drucker Prager plasticity", "true",
+                           Patterns::Bool (),
+                           "Whether to include Drucker-Prager plasticity in the rheological formulation.");
```

Comment:
```suggestion
                           "Whether to include Drucker-Prager plasticity in the composite rheological formulation.");
```

Same suggestion as above ...

Reply:
Makes sense. Done.